tests/test_graph.py

```diff
@@ -40,7 +50,10 @@ def setUp(self) -> None:
         self.audio_metadata = audio_meta_data(
             duration=200.0,
             sampling_rate=48000,
-            samples_count=200 * 48000)
+            samples_count=200 * 48000,
```

Comment:
Is this 200 the same as previous 200.0 one? What about 48000?

Reply:
I see no real difference here. Changed to `200.0 * 48000` for consistency. What's wrong with 48000 other than being hardcoded?

Reply:
Magic numbers.

Reply:
fixed